Mod0/computer-setup.md

```diff
@@ -0,0 +1,138 @@
+---
+layout:     page
+title:      Computer Setup
+permalink:  computer-setup
+---
+
+Complete the following steps before your first Mod 0 session in order to get the tools you'll need on your machine. Plan to spend 60-120 minutes on this.
+
+Here's what we'll cover in this guide. Click a link to jump to that portion of the guide:
+
+- [Install Chrome](#install-chrome)
+- [Install Slack](#install-slack)
+- [Create a GitHub Account](#create-a-github-account)
+- [Create a Replit Account](#create-a-replit-account)
+- [Create a Trello Account](#create-a-trello-account)
+- [Create a Loom Account](#create-a-loom-account)
+- [Install Zoom](#install-zoom)
+- [Snap your Windows](#snap-your-windows)
+
+<br>
+
+### Install Chrome
+
+Chrome includes a set of Developer Tools that will come in handy down the road. Additionally, it is always on the cutting edge of supporting new web technologies.
+
+**Action Steps:**
+- _If you haven't already,_ [download and install Chrome](https://www.google.com/chrome/).
+- Set Chrome as your default browser: 
+  - In your Chrome window, select the More menu by clicking on the three dots.
```

Comment:
This might be obvious, but maybe we should say that the dots are in the upper right corner?

Reply:
This was a copy/paste from the existing Mod 0 computer setup, but we can be more clear if necessary.